mortgageAPI.yaml

```diff
@@ -2608,6 +2608,19 @@ components:
               type: string
               description: Email of the applicant
               example: max.muster@domain.ch
+            rating:
+              type: integer
+              description: 'Solvency rating of the applicant e.g. CRIF'
+              example: '420'
```

Comment:
This is a number, not a string, so the single quotes need to be removed

Reply:
done, see below